dolly-backend-app/src/main/java/no/nav/dolly/bestilling/service/DollyBestillingService.java

```diff
@@ -343,4 +341,27 @@ protected void sendIdenterTilTPS(List<String> environments, List<String> identer
             tpsfResponseHandler.setErrorMessageToBestillingsProgress(e, progress);
         }
     }
+
+    protected Optional<DollyPerson> prepareDollyPersonTpsf(Bestilling bestilling,
+                                                         BestillingProgress progress) throws JsonProcessingException {
+
+        DollyPerson dollyPerson = null;
+        if (progress.isTpsf()) {
+            List<Person> personer = tpsfService.hentTestpersoner(List.of(progress.getIdent()));
+            if (!personer.isEmpty()) {
+                dollyPerson = dollyPersonCache.prepareTpsPersoner(personer.get(0));
+                sendIdenterTilTPS(Stream.of(bestilling.getMiljoer().split(",")).collect(Collectors.toList()),
```

Comment:
Hva med flytte ut denne til en egen funsjon slik at de ikke er så mange lister i denne funsjoen? 

`Stream.of(bestilling.getMiljoer().split(",")).collect(Collectors.toList()`
```java
private static List<String> getMiljoListe(Bestilling bestilling) {
   return Arrays.asList(bestilling.getMiljoer().split(","))
}
```